service/integration_test.go

```diff
@@ -502,7 +502,7 @@ func (t *TestSuite) TestStoreAndGetWithFilename() {
 	t.checkFile(t.url+path2+"?download_raw", path2, &t.noRole, 9, "", []byte("foobarbaz"))
 }
 
-func (t *TestSuite) TestStoreAndGetNodeAsAdminWithFormatAndTrailingSlash() {
+func (t *TestSuite) TestStoreAndGetNodeAsAdminWithFormatAndTrailingSlashAndSeekAndGet() {
```

Comment:
I'm surprised the linter didn't call this out as a run on sentence.

Reply:
Blame Go for not allowing commas in function names